Documentation/README_Interactable.md

```diff
@@ -0,0 +1,285 @@
+# Interactable
+![Interactable](/External/ReadMeImages/Interactable/InteractableExamples.png)
+
+## How to use Interactable
+Add the Interactable Component to a GameObject
+
+![Interactable](/External/ReadMeImages/Interactable/InteractableInspector_basicSteps.png)
+
+1. A Collider must exist on the GameObject with the Interactable or the child of the Interactable for it to receive input.
+2. Use the OnClick event to make something happen.
+3. Add visual feedback by linking a Target to a Profile and assigning a Theme.
+
+
+## Input Settings
+The basic features allow for button style interactions, such as pointer focus and clicks, that maps to interaction states to drive themes which are setup through the Interactable profile. Controller or hand focus, down, up and click (both near and far) are handled. Functionality can be extended using external scripts that can set state manually.
+
+<img src="/External/ReadMeImages/Interactable/InputFeatures_short.png" width="450">
+
+### Input Actions
+Select the action, from the input configuration or controller mapping profile, that the Interactable should react to.
+See the InputProfile or the DefaultMixedRealityInputActionsProfile for more on how Input Actions are setup and intended to be used in the application.
+ 
+### Enabled
+Sets the Interactable's enabled state, which will disable some input handling and update the themes to reflect the current state which is disabled.
+This is different from disabling input all together (using Enable Input) This means we have a button that would normally be interactive, but at this moment it is disabled (could be like a submit button waiting for all the required fields to be completed) and has a visual look and feel to denote it's disabled state.
+ 
+### IsGlobal
+Focus is not required to detect input actions, default behavior is false.
```

Comment:
Is there an example of what IsGlobal could accomplish (i.e. this allows for voice to work globally?)

Reply:
IsGlobal only refers to input like down, up and click. Does the object require focus for it to handle those events. Voice Commands being "global" is handled with the Requires Focus value under Voice Commands. Requires Focus only shows up when a valid string is entered into the Voice Commands field. (the two things are similar, but handle different use cases)